package ExceptionHandlingDemo;

public class ExceptionHandling3 {

	public static void main(String[] args) {
		{
	        int x[]=new int[10];
	        try
	        {
	            int y=5/0;
	            x[12]=100;

	        }
	        catch (ArithmeticException r)
	        {
	            System.out.println(r);
	        }
	        catch (ArrayIndexOutOfBoundsException c)
	        {
	            System.out.println(c);
	        }


	}

}
